packages/EasyCodingStandard/config/common/spaces.yml

```
@@ -37,9 +37,6 @@ checkers:
     # /**    @var   Type   $variable */
     PhpCsFixer\Fixer\Phpdoc\PhpdocSingleLineVarSpacingFixer: ~
 
-    # !$value;   =>   ! $value;
-    PhpCsFixer\Fixer\Operator\NotOperatorWithSuccessorSpaceFixer: ~
-
```

Duplicated and silently passed thanks to poo custom DI simulation, who knew :)